articles/client-platforms/ember2js/01-login.md

```diff
@@ -17,10 +17,10 @@ This tutorial and seed project have been tested with the following:
 :::
 
 <%= include('../../_includes/_package', {
-  pkgRepo: 'auth0-ember-simple-auth',
```

Comment:
@mehreencs87 Please add the following line:
`githubUrl: 'https://github.com/auth0-samples/auth0-emberjs2-sample/tree/master/00-Starter-Seed'`

See the updated [contributing guidelines](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#seed-projects) for details.